test/sample_tests.js

```diff
@@ -20,18 +20,19 @@ describe("# Sample boilerplate tests", () => {
         });
```

Comment:
This file could be probabbly removed by now, right?

Reply:
Out of the scope of this PR, imo

Leaving it for #19 - that would be more inside scope and there will be a better space for discussion and thought on the removal of it